ui/components/app/wallet-overview/index.scss

```diff
@@ -46,31 +46,29 @@
     align-items: center;
     margin: 16px 0;
     padding: 0 16px;
-    max-width: 100%;
+    max-width: 326px;
```

Comment:
This change scared me at first but large balances still look good in full screen mode.